src/screens/SignIn.js

```diff
@@ -149,8 +148,6 @@ export default class SignIn extends Component {
 					<Button text='Create Account' onPress={() => this._handleCreateAccount()} />
 				</View>
 
-				<KeyboardSpacer/>
```

Comment:
This should also be put back as we discussed

Reply:
Sorry, I had committed but forgot to push. Changes are all here now.